app/models/post.rb

```diff
@@ -1,3 +1,15 @@
 class Post < ApplicationRecord
-  belongs_to :author
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  after_create :update_user_posts_counter
+
+  def update_user_posts_counter
+    author&.update(posts_counter: author.posts.count)
+  end
```

Comment:
- [optional] It will be fine if you can make this method to be private since you only need to call it within this class. 👍

Reply:
Hello @OyePriscilla , I understand but I was advised to make it public since it is not explicitly required to make it private. I assume leaving the method without any access modifier would still be okay?